earthfile2llb/converter.go

```diff
@@ -1143,6 +1148,22 @@ func (c *Converter) Import(ctx context.Context, importStr, as string, isGlobal,
 	return c.varCollection.Imports().Add(importStr, as, isGlobal, currentlyPrivileged, allowPrivilegedFlag)
 }
 
+// Cache handles a `CACHE` command in a Target.
+// It appends run options to the Converter which will mount a cache volume in each successive `RUN` command.
+func (c *Converter) Cache(ctx context.Context, path string, isPersisted bool) error {
+	if err := c.checkAllowed(cacheCmd); err != nil {
+		return err
+	}
+	if isPersisted {
+		c.hasPersistentCache = true
```

Comment:
will this work with multiple caches? perhaps with a mix of one that persists and another that doesn't?

Reply:
> looking good, but I think we need to expand it to support (and test for) multiple caches.

Thanks, I did not think about the multiple cache case. Update on the way.